src/main/js/main_html5.js

```diff
@@ -1163,7 +1163,12 @@ require("../../../html5-common/js/utils/environment.js");
     var convertToSafeSeekTime = function(time, duration) {
       // If seeking within some threshold of the end of the stream, seek to end of stream directly
       if (duration - time < OO.CONSTANTS.SEEK_TO_END_LIMIT) {
-        time = duration;
+        // iOS has a bug when seeking to the exact end of a video, the player will lock up and become unresponsive.
+        if (OO.isIos) {
+          time = (duration - 0.01);
```

Comment:
Silly :)
But, good point! You need to make sure that time is not going to negative.

Oh, couple lines below this has negative number check.